doc/rule-descriptions.md

```diff
@@ -62,10 +62,11 @@
 
 ## WCAG 2.1 Level A & AA Rules
 
-| Rule ID                                                                                                            | Description                                                                                   | Impact  | Tags                         | Issue Type |
-| :----------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------- | :------ | :--------------------------- | :--------- |
-| [autocomplete-valid](https://dequeuniversity.com/rules/axe/3.5/autocomplete-valid?application=RuleDescription)     | Ensure the autocomplete attribute is correct and suitable for the form field                  | Serious | cat.forms, wcag21aa, wcag135 | failure    |
-| [avoid-inline-spacing](https://dequeuniversity.com/rules/axe/3.5/avoid-inline-spacing?application=RuleDescription) | Ensure that text spacing set through style attributes can be adjusted with custom stylesheets | Serious | wcag21aa, wcag1412           | failure    |
+| Rule ID                                                                                                            | Description                                                                                                | Impact  | Tags                         | Issue Type |
+| :----------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------- | :------ | :--------------------------- | :--------- |
+| [autocomplete-valid](https://dequeuniversity.com/rules/axe/3.5/autocomplete-valid?application=RuleDescription)     | Ensure the autocomplete attribute is correct and suitable for the form field                               | Serious | cat.forms, wcag21aa, wcag135 | failure    |
+| [avoid-inline-spacing](https://dequeuniversity.com/rules/axe/3.5/avoid-inline-spacing?application=RuleDescription) | Ensure that text spacing set through style attributes can be adjusted with custom stylesheets              | Serious | wcag21aa, wcag1412           | failure    |
+| [flag-role](https://dequeuniversity.com/rules/axe/3.5/flag-role?application=RuleDescription)                       | Flags elements whose role is none or presentation and which cause the role conflict resolution to trigger. | Minor   | cat.aria                     | failure    |
```

Comment:
The rule name is a bit vague. Ideally it would tell me what the rule is checking, like `button-name` or `image-alt`. `flag-role` doesn't really tell me what the rule is checking.

Maybe something like `role-conflict`?